### ISSUE TITLE
Clicking on Lay On Hands fails to roll on Cleric sheet

### ISSUE THREAD

**Issue by gabecook** [foundryvtt-dcc/dcc#458]
Clicking on the Lay on Hands label to roll fails. Same for Turn Unholy and Divine Aid.

![image](https://github.com/user-attachments/assets/178b460c-96d9-42cf-9369-f47c8b347090)

The following errors are in the console:

```
Foundry VTT | Rendering RollModifierDialog
dcc.js:174 Foundry VTT | Retrieved and compiled template systems/dcc/templates/dialog-roll-modifiers.html
dcc.js:480 TypeError: Cannot read properties of null (reading 'system')
    at SpellResult.addChatMessage (spell-result.js:49:27)
    at async Object.processSpellCheck (dcc.js:410:7)
    at async DCCActor.rollSkillCheck (actor.js:691:7)
processSpellCheck @ dcc.js:480
await in processSpellCheck
rollSkillCheck @ actor.js:691
await in rollSkillCheck
_onRollSkillCheck @ actor-sheet.js:726
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
```

![image](https://github.com/user-attachments/assets/bf87b669-71d6-40ce-893b-6b4ec011917e)

**Comment by cyface** [comment]
Thank you for the detailed report! I have fixed this in the just-released v0.50.25.

**Comment by gabecook** [comment]
@cyface Thank you so much. I tested this and it is now working as expected.
